frontend/app/utils/env.server.ts

```diff
@@ -49,7 +49,7 @@ const serverEnv = z.object({
   INTEROP_API_BASE_URI: z.string().url().default('https://api.example.com'),
   CCT_API_BASE_URI: z.string().url().default('https://api.example.com'),
   SCCH_BASE_URI: z.string().url().default('https://www.example.com'),
-
+  FIRST_NATIONS_YES_TYPE_ID: z.string().trim().min(1).default('first-nations-yes'),
```

Comment:
**Nitpick:** This environment variable seems randomly placed. Can it be moved closer to where other specific codes/IDs are in this file?

Reply:
moved closer as requested